doc/source/whatsnew/v1.0.0.rst

```diff
@@ -50,6 +50,8 @@ including other versions of pandas.
 Enhancements
 ~~~~~~~~~~~~
 
+- :meth:`Dataframe.info` now shows line numbers for the columns summary (:issue:`17304`)
```

Comment:
can you make a small sub-section showing these changes. move to the api breaking section as this is a change in the output repr.

Reply:
I added the difference to the *Backwards incompatible API changes* section. Hope that is what you meant